app/controllers/api/v1/orders_controller.rb

```diff
@@ -9,9 +9,14 @@ def create
 
   def update
     order = Order.find(params[:id])
-    menu_item = MenuItem.find(params[:menu_item])
-    order.order_items.create(menu_item: menu_item)
-    render json: create_json_response(order)
+    if params[:activity]
+      order.update_attribute(:finalized, true)
```

Comment:
should have a check so that user is required to complete the order

Reply:
Done